scripts/seed/run_seed.sh

```
@@ -0,0 +1,122 @@
+set -euo pipefail
+
+COMMAND="${1:-run}"
+DATASET="${2:-integrated}"
+SCALE="${3:-smoke}"
+SEED="${4:-42}"
+
+ENV_FILE=".env"
+
+if [ -f "${ENV_FILE}" ]; then
+  set -a
+  source "${ENV_FILE}"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)